packages/core/utils/src/dml/helpers/create-mikro-orm-entity.ts

```diff
@@ -92,6 +125,11 @@ export function createMikrORMEntity() {
     MikroORMEntity: EntityConstructor<any>,
     field: PropertyMetadata
   ) {
+    if (SPECIAL_PROPERTIES[field.fieldName]) {
+      SPECIAL_PROPERTIES[field.fieldName](MikroORMEntity, field)
+      return
+    }
```

Comment:
thought: Should we either log or throw to inform the end user of this case?

Reply:
Naah, this is when we are generating Mikro ORM entities from DML. I think the exception should be when DML receives it as part of the schema.

Let me add it there.

Reply:
I dont think we have to log it as it will part of the documentation

Reply:
@adrien2p Throwing exception?

Reply:
I believe we can throw an exception only if the user provides it as well, as you mention